### ISSUE TITLE
"overlay-plane" module is not listed in "_module.py" file

### ISSUE THREAD

**Issue by afshinmessiah** [MGHComputationalPathology/highdicom#33]
In the list of iods ( _iod.py), "[overlay-plane](https://github.com/MGHComputationalPathology/highdicom/blob/61f91ef004ebdf851b5092bb963ef6c5378c7e2d/src/highdicom/_iods.py#L7665)" is listed as a module for pet-image. In [_module.py](https://github.com/MGHComputationalPathology/highdicom/blob/master/src/highdicom/_modules.py) file though, there is no such module as "overley-plane" to get the list of its attributes.

**Comment by afshinmessiah** [comment]
The complete list of missing modules:

1. module multi-frame-overlay missing from iod: nm-image                                    
2. module multi-frame-overlay missing from iod: rt-dose                                     
3. module multi-frame-overlay missing from iod: x-ray-angiographic-image                    
4. module multi-frame-overlay missing from iod: xrf-image                                   
5. module overlay-activation  missing from iod: color-softcopy-presentation-state           
6. module overlay-activation  missing from iod: grayscale-softcopy-presentation-state       
7. module overlay-activation  missing from iod: pseudo-color-softcopy-presentation-state    
8. module overlay-activation  missing from iod: xa-xrf-grayscale-softcopy-presentation-state
9. module overlay-plane         missing from iod: cr-image                                    
10. module overlay-plane       missing from iod: ct-image                                    
11. module overlay-plane       missing from iod: digital-intra-oral-x-ray-image              
12. module overlay-plane       missing from iod: digital-mammography-x-ray-image             
13. module overlay-plane       missing from iod: digital-x-ray-image                         
14. module overlay-plane       missing from iod: mr-image                                    
15. module overlay-plane       missing from iod: pet-image                                   
16. module overlay-plane       missing from iod: sc-image                                    
17. module overlay-plane       missing from iod: us-image                                    
18. module overlay-plane       missing from iod: vl-endoscopic-image                         
19. module overlay-plane       missing from iod: vl-microscopic-image                        
20. module overlay-plane       missing from iod: vl-photographic-image                       
21. module overlay-plane       missing from iod: vl-slide-coordinates-microscopic-image

**Comment by hackermd** [comment]
These are all retired attributes containing `x` in the group or element part of the tag. pydicom stores those attributes not in the `pydicom._dicom_dict.DicomDictionary` but in a separate `pydicom._dicom_dict.RepeatersDictionary` and looking up the keyword corresponding to the tag requires a workaround.

Should be fixed with 6fd6e8ef67439cd7fa0f67724210bce10d79175b.

@afshinmessiah could you please confirm? It would also be great if you could add a few unit tests.

**Comment by afshinmessiah** [comment]
I checked the newly generated _iod and _module files. Some modules are missing: 

1. module **real-time-audio-waveform-current-frame-functional-groups**           missing from iod: **real-time-audio-waveform**          
2. module **real-time-video-endoscopic-image-current-frame-functional-groups**   missing from iod: **real-time-video-endoscopic-image**  
3. module r**eal-time-video-photographic-image-current-frame-functional-groups** missing from iod: **real-time-video-photographic-image**

I've not added unit-tests yet but I will soon.

**Comment by hackermd** [comment]
Yes, the attributes in Current Frame Functional Groups Module with group `0006` are skipped, since they are not part of the DICOM Dictionary.

We could handle that special case separate if needed. Out of curiosity, what do you need these attributes for?

**Comment by afshinmessiah** [comment]
No, I don't need them. I just ran a test on all iods and modules to see if all modules are included.

**Comment by hackermd** [comment]
> No, I don't need them. I just ran a test on all iods and modules to see if all modules are included.

Ok. Thanks for the effort. I can include them for completeness (by hardcoding `0006` attributes).

If you already have test procedures to evaluate completeness, it would be great if we could include them into the unit tests located in `/tests`.
